### DISCUSSION TITLE
SingleSignOn SSO

### DISCUSSION THREAD

**Discussion by Dtrieb** [thorsten/phpMyFAQ#2663] [Ideas]
Wäre es möglich einen SSO zum anmelden benutzen?

**Comment by thorsten** [comment]
Yes, it's possible

**Reply by Dtrieb** [reply]
With the actual version?

**Reply by thorsten** [reply]
Yes, for years it's possible. Since v3.2 you can use Azure AD as well.

**Comment by yann2-0** [comment]
Hi,
I was using phpmyfaq 3.0.8 with sso which worked very well, but now I'm using 3.2 and sso no longer works.
For information I send $_SERVER variables but it no longer works.
Have you already had any feedback?

**Reply by thorsten** [reply]
Hi @yann2-0 could you please add a new issue here to track the bug?
